functionalDB/functionalDB.markdown

```diff
@@ -668,14 +668,14 @@ Our data model is based on accumulation of facts (i.e., datoms) over time. For t
 ### Query Language

```

Comment:
a subset of the Datalog language from the => a subset of the Datalog-based query language from the